### PR TITLE
TMEDIA-586 - Remove export for blocks

### PR DESCRIPTION
**Pull request by matthewroach**
## Description

Blocks were never designed to be consumed via code only via PageBuilder, some blocks have exported code to be made available to other blocks, and this is an anti-pattern and not a supported way of using blocks.

## Jira Ticket
- [TMEDIA-586](https://arcpublishing.atlassian.net/browse/TMEDIA-586)

## Test Steps
- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout TMEDIA-586`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/ads-block,@wpmedia/byline-block,@wpmedia/date-block,@wpmedia/links-bar-block,@wpmedia/numbered-list-block,@wpmedia/overline-block,@wpmedia/placeholder-image-block,@wpmedia/results-list-block,@wpmedia/video-player-block`
3. Valid nothing is broken and current pages/templates are still working as expected

## Dependencies or Side Effects
Custom blocks could be relying on these imports which would cause issues in their code, the import would stop working. These imports have not been supported, and client would need to adjust their code to account for these changes

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
- [x] Confirmed all the test steps a reviewer will follow above are working.
- [x] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [x] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [x] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [x] Confirmed this PR has unit test files
  - [x] Ran `npm run test`, made sure all tests are passing
  - [x] If the amount of work to write unit tests for this change are excessive,
please explain why (so that we can fix it whenever it gets refactored).
- [x] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

### CODE REVIEW

File: blocks/placeholder-image-block/index.js
Comment (JackHowa) [inline blocks/placeholder-image-block/index.js:4]: you're keeping this one? I'm not sure I follow

Comment (JackHowa) [reply blocks/placeholder-image-block/index.js:4]: Looks like there may have been a reason for this? 

```
Two “blocks” that are on watch include: 

Global Phrases block [arc-themes-blocks/blocks/global-phrases-block at canary · WPMedia/arc-themes-blocks](https://github.com/WPMedia/fusion-news-theme-blocks/tree/canary/blocks/global-phrases-block) 

Placeholder block [arc-themes-blocks/blocks/placeholder-image-block at canary · WPMedia/arc-themes-blocks](https://github.com/WPMedia/fusion-news-theme-blocks/tree/canary/blocks/placeholder-image-block)
```

Comment (matthewroach) [reply blocks/placeholder-image-block/index.js:4]: Yeah - if you look at the ticket it's documented why these need to be kept

Comment (JackHowa) [reply blocks/placeholder-image-block/index.js:4]: ah didn't see it deep in the comments

**Review by JackHowa — DISMISSED** [review]
Seeing some errors around ads-block. if you go to a gallery that has ads in it, then scroll to the ad, you will see the error http://localhost/pf/all-blocks/?_website=the-sun

```
Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Check the render method of `AdBlock`.
```

**Review by JackHowa — COMMENTED** [review]
ads working as expected -- sorry for the hassle on this one 
Screen Shot 2022-02-09 at 09 45 52

**Review by JackHowa — APPROVED** [review]
oops approved

**Comment by matthewroach** [comment]
Awaiting feedback from product before we merge this - it may get held up based on approach we want to take with giving clients a heads up on the changes coming

**Comment by JackHowa** [comment]
do not merge this -- awaiting product go-ahead

**Comment by RobinGiannattasio** [comment]
NERD ALERT NERD ALERT!

I love using PR labels :D

**Comment by JackHowa** [comment]
hahah at risk of spamming Matthew, yeah I just like the harsh red color of 'Do not merge' as well -- feel like it matches the intent

**Comment by RobinGiannattasio** [comment]
Haha, well, even thought we are spamming Matthew - one real comment!

We should make sure to be cognizant of changing the base branch when we *are* ready to merge, since there will be a new release that this is associated with.

**Pull request merged**
